com.unity.multiplayer.mlapi/CHANGELOG.md

```diff
@@ -1,14 +1,89 @@
 # Changelog
-All notable changes to this package will be documented in this file. The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+This file documents all notable changes to this package. Additional documentation and release notes are available at [Multiplayer Documentation](https://docs-multiplayer.unity3d.com).
 
-## [0.1.0] - 2020-03-02
-This is the initial experimental Unity MLAPI Package.
-### Changed
-- Rather than performance / convenience RPCs, we now have one performant 'Standard' RPC type
-- RPCs are now deferred and queued to support upcoming features and improve bandwidth usage
-- A preliminary tick system has been added
-- Network variables are tick-based
+## [0.1.0] - 2021-03-23
+
+This is the initial experimental Unity MLAPI Package, v0.1.0.
+
+### New Features
+
+- Refactored a new standard for Remote Procedure Call (RPC) in MLAPI which provides increased performance, significantly reduced boilerplate code, and extensibility for future-proofed code. MLAPI RPC includes `ServerRpc` and `ClientRpc` to execute logic on the server and client-side.
```

Comment:
I actually think for user clarity, and since this is a changelog after all, the verbiage explaining how we formerly had perf / conv RPCs and now just have a single performant unified RPC solution should remain.

Reply:
I have info in the removed features on removed conv/perf, can note it in this entry with a slight tweak so it's not redundant :)